rss_pipeline/requirements.txt

```diff
@@ -1,3 +1,6 @@
+beautifulsoup4
+lxml==4.9.3
```

Comment:
Given this is the only library where a specific version is being used, it would be useful to document this in the `README.md`